Dockerfile

```diff
@@ -61,6 +61,12 @@ wget -P /build/software/maven https://archive.apache.org/dist/maven/maven-2/2.2.
 
 RUN mkdir -p /build/software/java
 
+COPY amazon-corretto-11.0.4.11.1-linux-x64.tar.gz /build/software/java
```

Comment:
Since we can direct download corretto from amazon, I suggest we wget this.

https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/downloads-list.html

Reply:
Done